### ISSUE TITLE
Show error message while submitting when user doesn't participate in a challenge.

### ISSUE THREAD

**Issue by RishabhJain2018** [Cloud-CV/evalai-cli#106]
Error message like you have not participated in the challenge is not shown when a user tries to submit to a challenge without participating in it.

**Comment by pydevsg** [comment]
Hi Rishabh how can I help you ?
whats your issue with the code, code you specify?

**Comment by isht3** [comment]
@pydevsg Hey there! Have you set up the development environment of the CLI? If yes, try running the submission command without participating in a challenge and we'll get a very vague response, we need to change it to something a bit more verbose.

**Comment by gautamjajoo** [comment]
@RishabhJain2018 Is this the error message
``404 Client Error: Not Found for url: http://localhost:8000/api/jobs/submission/1 ``

**Comment by mahim23** [comment]
@RishabhJain2018 @isht3 Can I take this issue up if it still needs work?

**Comment by RishabhJain2018** [comment]
yeah, sure. but since you're working on another issue, it is open for others to work on.

**Comment by PrathyushaModala** [comment]
Hi @RishabhJain2018 ,
Is the issue still open, I would Like to work on it??
